src/Splat.AppCenter/Splat.AppCenter.csproj

```diff
@@ -1,7 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net5.0-windows10.0.17763.0;net6.0-windows10.0.17763.0</TargetFrameworks>
```

Comment:
I believe should be 19041 minimum

Reply:
i'd agree but appcenter targets 17763 :). down to 1 test failure. sorting a vm with vs2022 on it so i can stop throwing things at the build server